src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,12 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
-    <Project Include="System.Private.Xml.csproj">
+    <!--<Project Include="System.Private.Xml.csproj">
```

Comment:
Please correct me if I'm wrong. I believe we don't need these because all the Xml packages for netcore50 and netcore50aot are not using System.Private.Xml and will come from package harvesting anyway. So, there is no point in building this package for these configurations.